packages/salesforcedx-vscode-apex/src/languageClientUtils/javaDocSymbols.ts

```diff
@@ -0,0 +1,48 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Moved to a utility and added the OSS header we normally use in this type of scenario.